mw_endpoint.py

```diff
@@ -43,17 +45,20 @@ def inject_config_qrcode_svg():
 #
 
 @app.route("/config")
+@limiter.exempt
```

Comment:
why are you exempting all the endpoints of this module?

Reply:
i'll take them out and leave it to the mw_endpoint limiter which is limited to 100/hour. The issue is if you happen to have multiple users make more than the mw_endpoint threshold the user will get error when they try to do anymore.